index.js

```diff
@@ -3,10 +3,22 @@
 //
 
 const debug = require('debug');
-const server = require('http').createServer();
-const io = require('socket.io')(server);
+const { createServer } = require('http');
+const socketIO = require('socket.io');
+const micro = require('micro')
 
-require('./server')({ io });
+const { SignalSwarmServer } = require('@geut/discovery-swarm-webrtc/server');
+
+let signal
+const server = createServer(micro(async (req, res) => {
+  const result = {}
```

Comment:
```
const { version } = require('./package.json') // does this mean babel?
result = {
  version: xxx
  channels: []
}
```

Reply:
no, you can do that without the need of babel if you want. `require('./package.json')` will work